src/components/admin/feedback/ManageFeedback.jsx

```
@@ -34,6 +34,7 @@
     const fetchAllFeedbacks = async () => {
         try {
             const data = await api.get("customer-feedbacks")
+            const data = await api.get("customer-feedbacks")
```

Biome (1.9.4)

[error] Shouldn't redeclare 'data'. Consider to delete it or rename it.

'data' is defined here:


(lint/suspicious/noRedeclare)